lib/main.coffee

```diff
@@ -49,8 +38,8 @@ module.exports =
         filePath = textEditor.getPath()
         command = @executablePath
         ruleset = @rulesets
-        if @projectRules
-          rulesetPath = helpers.find(filePath, 'ruleset.xml')
+        if /^[a-z0-9]+\.xml$/gi.test(@rulesets)
```

Comment:
Thanks for the detailed description. This will have to be a breaking change since the expected behavior will be different from before, but I agree with your reasoning here.

And I should remember to test stuff like that instead of just looking at the code :stuck_out_tongue:.